### PR TITLE
feat:add the function of sql parsing

### PR DESCRIPTION
**Pull request by coryhh**

### CODE REVIEW

File: src/main/java/com/arextest/schedule/comparer/sqlparse/SqlParseUtil.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+    public static ObjectNode sqlParse(String sql) {
+        ObjectNode sqlObj = JsonNodeFactory.instance.objectNode();
+        try {
+            Statement statement = CCJSqlParserUtil.parse(sql);
+            Parse parse = ActionFactory.selectParse(statement);
+            sqlObj = (ObjectNode) parse.parse(statement);
+        } catch (Throwable throwable) {
+            sqlObj.put(ORIGINAL_SQL, sql);
```
Comment (mokdeve) [inline src/main/java/com/arextest/schedule/comparer/sqlparse/SqlParseUtil.java:24]: Add warning log when exception occurs

File: src/main/java/com/arextest/schedule/comparer/sqlparse/action/ActionFactory.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.arextest.schedule.comparer.sqlparse.action;
+import com.arextest.schedule.comparer.sqlparse.Parse;
+import net.sf.jsqlparser.statement.Statement;
+
+/**
+ * Created by rchen9 on 2023/1/6.
+ */
+public class ActionFactory {
+    public static Parse selectParse(Statement statement) {
+        String simpleName = statement.getClass().getSimpleName();
```
Comment (mokdeve) [inline src/main/java/com/arextest/schedule/comparer/sqlparse/action/ActionFactory.java:10]: >  String simpleName = statement.getClass().getSimpleName();
 don't use class simpleName for switch case, The name maybe change.

File: src/main/java/com/arextest/schedule/comparer/sqlparse/SqlParseUtil.java (hunk 1 above)
Comment (coryhh) [reply src/main/java/com/arextest/schedule/comparer/sqlparse/SqlParseUtil.java]: done

File: src/main/java/com/arextest/schedule/comparer/sqlparse/action/ActionFactory.java (hunk 2 above)
Comment (coryhh) [reply src/main/java/com/arextest/schedule/comparer/sqlparse/action/ActionFactory.java]: done

**Pull request closed without merging**